### PR TITLE
Add pillow

### PR DESCRIPTION
**Pull request by Qubad786**
Add pillow, its required for ImageField

### CODE REVIEW

**Comment by Qubad786** [comment]
@muhammad-ammar, @muzaffaryousaf  have a look.

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ lxml==3.3.6
 django-storages==1.5.2
 boto==2.46.1
 django-model-utils==2.3.1
+Pillow==4.2.1
```
Comment (muhammad-ammar) [inline requirements.txt]: @Qubad786 edx-platform is uses [Pillow==3.4](https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt#L83)

**Comment by Qubad786** [comment]
is it good to merge? @muhammad-ammar , @muzaffaryousaf ^

**Comment by muhammad-ammar** [comment]
👍

**Pull request merged**